py-polars/tests/unit/test_functions.py

```diff
@@ -130,8 +130,8 @@ def test_null_handling_correlation() -> None:
 
     out = df.select(
         [
-            pl.pearson_corr("a", "b").alias("pearson"),
-            pl.spearman_rank_corr("a", "b").alias("spearman"),
+            pl.corr("a", "b").alias("pearson"),
+            pl.corr("a", "b", "spearman").alias("spearman"),
```

Comment:
We should call method as a keyword argument, i.e. `corr("a", "b", method="spearman")`

Reply:
Now it´s a keyword argument.